include/private/soci-mktime.h

```diff
@@ -21,6 +21,24 @@ namespace soci
 namespace details
 {
 
+namespace inner 
```

Comment:
Do we really need this namespace? `details` is already a kind of "inner" namespace.

Reply:
fixed